DataRepo/loaders/msruns_loader.py

```
@@ -1869,24 +1904,57 @@ def get_msrun_sequence_from_dir(
             if default_msrun_sequence is not None:
                 self.aggregated_errors_object.buffer_warning(
                     MzxmlNotColocatedWithAnnot(
-                        file=os.path.join(mzxml_dir, mzxml_name),
+                        annot_dirs=list(self.annotdir_to_seq_dict.keys()),
+                        file=os.path.join(mzxml_dir, mzxml_filename),
                         suggestion=f"Using the default sequence '{default_msrun_sequence.sequence_name}'.",
                     )
                 )
                 msrun_sequence = default_msrun_sequence
+            else:
+                self.aggregated_errors_object.buffer_error(
+                    MzxmlNotColocatedWithAnnot(
+                        annot_dirs=list(self.annotdir_to_seq_dict.keys()),
+                        file=os.path.join(mzxml_dir, mzxml_filename),
+                        suggestion=(
+                            "Either fill in default sequences for the peak annotation files in the "
+                            f"'{PeakAnnotationFilesLoader.DataSheetName}' sheet or move "
+                            f"'{os.path.join(mzxml_dir, mzxml_filename)}' and its peak annotation file(s) into a "
+                            "common directory (and don't forget to update path in the "
+                            f"'{PeakAnnotationFilesLoader.DataHeaders.FILE}' column in the "
+                            f"'{PeakAnnotationFilesLoader.DataSheetName}' sheet)."
+                        ),
+                    )
+                )
```

C. If we don't have a default sequence, we need to raise a fatal error.  This is in addition to the warning above when we DO have a default sequence.